openpower/package/bonnell-xml/bonnell-xml.mk

```diff
@@ -0,0 +1,214 @@
+            --bios-schema-file=$(BONNELL_XML_BIOS_SCHEMA_FILE) \
+            --bios-output-file=$(BONNELL_XML_BIOS_METADATA_FILE)
+
+        # Transform BIOS XML into Petitboot specific BIOS XML via the schema
+        xsltproc -o \
+            $(BONNELL_XML_PETITBOOT_BIOS_METADATA_FILE) \
+            $(BONNELL_XML_PETITBOOT_XSLT_FILE) \
+            $(BONNELL_XML_BIOS_METADATA_FILE)
+
+        # Create the wofdata
+        if [ -e $(BONNELL_XML_MRW_HB_TOOLS)/$(WOF_TOOL) ]; then \
+            chmod +x $(BONNELL_XML_MRW_HB_TOOLS)/$(WOF_TOOL); \
+        fi
+
+        # Create WOF override image
+	$(eval WOF_OVERRIDE_BIN = $$(patsubst %.xml,%.wofdata,$(call qstrip,$(BR2_BONNELL_XML_FILENAME))))
+	if [ -e $(BONNELL_XML_MRW_SCRATCH)/$(WOF_SETS_TAR) ]; then \
+	    rm $(BONNELL_XML_MRW_SCRATCH)/$(WOF_OVERRIDE_BIN); \
+	    cd $(BONNELL_XML_MRW_SCRATCH) && mkdir $(WOF_SETS_DIR); \
+            cd $(BONNELL_XML_MRW_SCRATCH)/$(WOF_SETS_DIR) && tar -xzvf $(BONNELL_XML_MRW_SCRATCH)/$(WOF_SETS_TAR); \
+            cd $(BONNELL_XML_MRW_SCRATCH)/$(WOF_SETS_DIR)/WofSetBins && ls | grep -i "\.bin" > $(WOF_BIN_OVERRIDE_LIST); \
+	    cd $(BONNELL_XML_MRW_SCRATCH)/$(WOF_SETS_DIR)/WofSetBins && $(BONNELL_XML_MRW_HB_TOOLS)/$(WOF_TOOL) --create $(BONNELL_XML_MRW_SCRATCH)/$(WOF_OVERRIDE_BIN) --combine $(BONNELL_XML_MRW_SCRATCH)/$(WOF_SETS_DIR)/WofSetBins/$(WOF_BIN_OVERRIDE_LIST); \
+	else \
+	    cd $(BONNELL_XML_MRW_SCRATCH) && dd if=/dev/zero of=$(WOF_OVERRIDE_BIN) bs=4096 count=1; \
+	fi
+
+        # Create SPD image, if necessary
+        $(eval PSPD_BIN = $$(patsubst %.xml,%.PSPD.bin,$(call qstrip,$(BR2_BONNELL_XML_FILENAME))))
+        $(eval SPD_IMAGE_FILES = $$(wildcard $(BONNELL_XML_MRW_SCRATCH)/*.pspddata*))
+
+        # Cleanup old PSPD.bin
+        if [ -e "$(BONNELL_XML_MRW_SCRATCH)/$(PSPD_BIN)" ]; then \
+            rm $(BONNELL_XML_MRW_SCRATCH)/$(PSPD_BIN); \
+        fi
+
+        $(foreach spd_image_file, $(SPD_IMAGE_FILES), \
+            $(call build_ecImgs,$(spd_image_file),$(IMAGE_TOKEN)))
+
+        # SPD_FILES_TOKENED is what is used as input for the --ecImg_'s to buildSPDImages.pl
+        #
+        # If SPD_FILES_TOKENED is empty we get zero SPD Images built (tocCount=0)
+        #
+        # SPD_ARG_ITEMS (will only output debug info if buildSPDImages.pl will result in tocCount more than zero)
+        $(foreach spd_arg, $(SPD_FILES_TOKENED), echo SPD_ARG_ITEMS $(spd_arg);)
+
+        # If we have SPD images the PSPD.bin will be built containing the data, otherwise we get PSPD.bin with all FF's
+        if [ -e "$(BONNELL_XML_MRW_HB_TOOLS)/buildSPDImages.pl" ] && [ -n "$(SPD_IMAGE_FILES)" ]; then \
+            chmod +x $(BONNELL_XML_MRW_HB_TOOLS)/buildSPDImages.pl; \
+            echo "*** RUNNING buildSPDImages.pl ***";\
+            $(BONNELL_XML_MRW_HB_TOOLS)/buildSPDImages.pl --spdOutBin $(BONNELL_XML_MRW_SCRATCH)/$(PSPD_BIN) $(SPD_FILES_TOKENED); \
+        fi
+
+        # Create the MEMD binary
```

Comment:
Just a note that we don't have MEMD in P10 so this should probably disappear from all of our mk files...  (no need to make a change here though)

Reply:
Thank you for this note.  I will remove next week in another PR